integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/MonitoringUtils.java

```diff
@@ -848,7 +848,7 @@ public static void verifyMonExpAppAccessThroughNginx(String nginxHost, int repli
             ADMIN_PASSWORD_DEFAULT,
             K8S_NODEPORT_HOST,
             nodeport);
-    assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, managedServerNames, 50))
+    assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, managedServerNames, 100))
```

Comment:
Instead of simply increasing the iteration can we use the testUntil to wait for the required response?

Reply:
Please remove the commented code.